apiserver/plane/api/serializers/page.py

```diff
@@ -102,6 +80,41 @@ def update(self, instance, validated_data):
         return super().update(instance, validated_data)
 
 
+class SubPageSerializer(BaseSerializer):
+    entity_details = serializers.SerializerMethodField()
+
+    class Meta:
+        model = PageLog
+        fields = "__all__"
+        read_only_fields = [
+            "workspace",
+            "project",
+            "page",
+        ]
+
+    def get_entity_details(self, obj):
+        entity_name = obj.entity_name
+        if entity_name == 'forward_link' or entity_name == 'back_link':
+            try:
+                page = Page.objects.get(pk=obj.entity_identifier)
+                return PageSerializer(page).data
+            except Page.name:
```

Comment:
@NarayanBavisetti can you check this this should be `Page.DoesNotExist`

Reply:
Yes. Made the change